python/paddle/fluid/reader.py

```diff
@@ -248,12 +296,29 @@ def __next__(self):
                 return self.next()
 
             def next(self):
-                ret = self._reader.read_next()
-                if ret:
-                    return ret
+                if not in_dygraph_mode():
+                    ret = None
```

Comment:
Not necessary to set `ret=None` beforehand.

Reply:
ok